spacy/tests/regression/test_issue4725.py

```diff
@@ -0,0 +1,25 @@
+# coding: utf8
+from __future__ import unicode_literals
+
```

Comment:
I think we can drop these two lines on `develop`, right? (same comment for all the new code files in this PR)

Reply:
Ah, didn't realise that so many of those were left. Fixing.